src/LookPage/index.js

```diff
@@ -75,57 +74,56 @@ const ButtonContainer = styled.div`
 `
 
 class LookPage extends Component {
-  state = {
-    looks: [
-      {
-        id: 2,
-        image: 'https://i.imgur.com/iKT9fl6.jpg',
-        brands: ['Ralph Lauren', 'Armani'],
-        description: 'The description',
-      },
-      {
-        id: 3,
-        image: 'https://i.imgur.com/LPPxE3J.png',
-        brands: ['Ralph Lauren', 'Armani'],
-        description: 'The description',
-      },
-    ],
-    currentLook: {
-      id: 1,
-      image: 'https://i.imgur.com/n1IqG2c.jpg',
-      brands: ['Ralph Lauren', 'Armani'],
-      description: 'The description',
-    },
-  }
+  // state = {
```

Comment:
I think it's safe to delete the commented out code now since it also exists in App?

Reply:
This fails tests - is this intentional?